### ISSUE TITLE
Xamarin Android Designer Does Not Detect Changes Unless Switch to Source Tab Made

### ISSUE THREAD

**Issue by novatekinno** [xamarin/xamarin-android#1722]
### Steps to Reproduce

0. Install current version of Visual Studio Community & Mobile Development with .NET tools. All default selections.
1. Create a Single View Android App
2. Open the content_main.axml layout in the Designer
3. Drag in a Text (Medium) control.
4. Notice the file name tab above does not show an asterisk for an unsaved change.
5. Press the Source tab below the design view. Notice there is now an unsaved change detected.

### Expected Behavior
Changes in the design view will show unsaved changes. Closing the design view without saving will prompt the user to save.

### Actual Behavior
Changes in design view will not be persisted when closing unless the user tabs over to the Source view and then saves and closes the file.

### Version Information
Microsoft Visual Studio Community 2017 
Version 15.7.2
VisualStudio.15.Release/15.7.2+27703.2018
Microsoft .NET Framework
Version 4.7.02556

Installed Version: Community

C# Tools   2.8.2-beta6-62916-08. Commit Hash: 2ad4aabc7a9dada097e54e544ebba48ab1c05074
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

Merq   1.1.19-rc (a4ffc1b)
Command Bus, Event Stream and Async Manager for Visual Studio extensions.

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Mono Debugging for Visual Studio   4.10.5-pre (ab58725)
Support for debugging Mono processes with Visual Studio.

NuGet Package Manager   4.6.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

ResourcePackage Extension   1.0
ResourcePackage Visual Studio Extension Detailed Info

Visual Basic Tools   2.8.2-beta6-62916-08. Commit Hash: 2ad4aabc7a9dada097e54e544ebba48ab1c05074
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools 10.1 for F# 4.1   15.7.0.0.  Commit Hash: 56f97a8dd4353d897336941f3e644423b87c794f.
Microsoft Visual F# Tools 10.1 for F# 4.1

Visual Studio Code Debug Adapter Host Package   1.0
Interop layer for hosting Visual Studio Code debug adapters in Visual Studio

VisualStudio.Mac   1.0
Mac Extension for Visual Studio

Xamarin   4.10.0.448 (4373404db)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin Designer   4.12.270 (82d750d12)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin.Android SDK   8.3.0.19 (HEAD/342b2ce96)
Xamarin.Android Reference Assemblies and MSBuild support.

Xamarin.iOS and Xamarin.Mac SDK   11.10.1.178 (408d357)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.

**Comment by Aufty** [comment]
Can I upvote this? I swear it was fixed but it looks like it it's happening again

**Comment by JonDouglas** [comment]
@Aufty Yes! You can use a github reaction to the top level post! Do you perhaps have your Visual Studio version information handy? This should already be fixed.

**Comment by JonDouglas** [comment]
@Aufty I understand your frustrations. We have been hard at work to revamp this experience which we blogged about here: https://blog.xamarin.com/future-xamarin-android-designer/

We introduced a split view in the latest preview that we hope resolves this workflow issue. If you do give it a try, we would love your feedback on it!
